tui/installer/screens.py

```diff
@@ -482,10 +485,14 @@ def confirm_erase_volume_groups(answers):
     return RIGHT_FORWARDS
 
 def use_extra_media(answers):
+    message = "You installed Would you like to install any Supplemental Packs?"
+    if "driver-repos" in answers:
+        message += ("  You loaded one or more Driver Disks during this installation, "
+                    "you likely want to install them now using this feature.")
```

Comment:
Would this be better as "You previously loaded one or more Driver Disks, if you wish to include these drivers on the installed system, you must install them now"?

Reply:
Fine with me, will reroll.